pkg/serviceimport/map.go

```diff
@@ -1,9 +1,9 @@
-package multiclusterservice
+package serviceimport
 
 import (
 	"sync"
 
-	lighthousev1 "github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v1"
+	lighthousev2a1 "github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v2alpha1"
```

Comment:
I'd like to see this moved from plugin to agent. The agent should aggregate ServiceImports from multiple clusters into single object and plugin uses it. It will allow us to be compatible with u/s API and won't need changes when it becomes available in K8s.

Reply:
I can push it as another patch with those changes.